### PR TITLE
Configuration file read for endpoint

### PR DESCRIPTION
**Pull request by aburgosu**

### CODE REVIEW

File: config.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Web Service Convert EndPoint
+
+server.endpoint=http://127.0.0.1/convert
```
Comment (MelissaRoman06) [inline config.properties]: add empty line

File: src/main/java/com/fundation/logic/model/ServiceConnection.java [hunk 2]
```diff
@@ -39,11 +42,25 @@
     public static final int SUCCESS=1;
     public static final int ERROR=2;
 
-    public ServiceConnection(String uri) {
+    public ServiceConnection() {
         httpClient = HttpClients.createDefault();
+        String uri = readConfigurationEndpoint();
         httpPost = new HttpPost(uri);
     }
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/ServiceConnection.java:54]: Put documentation

File: src/main/java/com/fundation/logic/model/ServiceConnection.java [hunk 3]
```diff
@@ -39,11 +42,25 @@
...
 
+    private String readConfigurationEndpoint() {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/ServiceConnection.java:59]: add documentation

File: src/main/java/com/fundation/logic/model/ServiceConnection.java [hunk 4]
```diff
@@ -39,11 +42,25 @@
     public static final int SUCCESS=1;
     public static final int ERROR=2;
 
-    public ServiceConnection(String uri) {
+    public ServiceConnection() {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/ServiceConnection.java:49]: add documentation

**Pull request merged**